CinemaStar/DetailsMovies/ViewModel/DetailsMovieViewModel.swift

```diff
@@ -0,0 +1,53 @@
+// DetailsMovieViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Интерфейс вью моделя для экрана списка фильмов
+protocol MovieDetailViewModelProtocol {
+    /// Состояние данных
+    var state: ModelViewState<MovieDetail> { get set }
+    /// Получение фильма с сети
+    func fetchMovie(completion: @escaping VoidHandler)
+    /// Получение изображения фильма с сети
+    func backToMoviesList()
+}
+
+/// Вью модель для экрана списка фильмов
+final class MovieDetailViewModel: MovieDetailViewModelProtocol {
+    // MARK: - Public Properties
+
+    var state: ModelViewState<MovieDetail>
```

Comment:
В таком варианте любой у кого есть ссылка на MovieDetailsViewModel может перезаписать значение стейт. Лучше приватить сеттер, чтобы только viewModel управляла записью, а читать могли уже все
```suggestion
    private(set) var state: ModelViewState<MovieDetail>
```